packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -65,6 +66,17 @@ export default class UpdateObject extends BaseAction {
 				message: this.translationService.getMessage(ERRORS.NO_ACTIVE_WORKSPACE),
 			};
 		} else {
+			const projectInfoService = new ProjectInfoServive(this.executionPath);
+			if (projectInfoService.isAccountCustomizationProject() || projectInfoService.isSuiteAppProject) {
```

Comment:
The `new ProjectInfoService` expects a project folder as a parameter not the execution path, I'm unable to run the command properly in my vscode because I get an uncaught exception thrown by the `projectInfoService.isAccountCustomizationProject()` call.
When I did work on the vscode deploy, I had kind of the same problem, you can check there what I did, and probably we will need to extract to service that is accessible to for more commands (like deploy or more to come).

In the next line a couple of parenthesis is missing in `projectInfoService.isSuiteAppProject`.

Reply:
Updated.